collaborations/dark-tides-of-bilgewater/rogue-wildcard.xml

```diff
@@ -0,0 +1,152 @@
+<elements>
+	<info>
+		<update version="0.0.1">
+			<file name="rogue-wildcard.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/collaborations/dark-tides-of-bilgewater/rogue-wildcard.xml" />
+		</update>
+	</info>
+
+	<element name="Wild Card" type="Archetype" source="Legends of Runeterra: Dark Tides of Bilgewater" id="ID_DDB_LRDTOB_ARCHETYPE_WILD_CARD">
+		<supports>Roguish Archetype</supports>
+		<description>
+			<p>Some rogues favor honing their skill and precision, others perfect a silver tongue, but you—well, you’ve always gotten a kick out of leaving things up to chance.</p>
+			<p>Rogues of the Wild Card variety thrive on the high stakes and random thrill of the game table. Armed with cunning cheats and no small amount of luck, these sly gamblers live and die by their rules, never keen to simply accept the hand fate deals them.</p>
+			<div element="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_TRICKS_UP_THE_SLEEVE" />
+			<div element="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_WILD_CARDS_GAMBIT" />
+			<div element="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_SHIFTING_THE_ODDS" />
+			<div element="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_TWIST_OF_FATE" />
+			<div element="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_JOKER_WILD" />
+		</description>
+		<sheet display="false">
+			<description></description>
+		</sheet>
+		<rules>
+			<grant type="Archetype Feature" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_TRICKS_UP_THE_SLEEVE" level="3" />
+			<grant type="Archetype Feature" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_WILD_CARDS_GAMBIT" level="3" />
+			<grant type="Archetype Feature" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_SHIFTING_THE_ODDS" level="9" />
+			<grant type="Archetype Feature" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_TWIST_OF_FATE" level="13" />
+			<grant type="Archetype Feature" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_JOKER_WILD" level="17" />
+		</rules>
+	</element>
+
+	<element name="Tricks Up the Sleeve" type="Archetype Feature" source="Legends of Runeterra: Dark Tides of Bilgewater" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_TRICKS_UP_THE_SLEEVE">
+		<description>
+			<p>Your penchant for games has afforded you the ability to subtly manipulate fortune to your favor. When you choose this archetype at 3rd level, you learn the guidance cantrip. Starting at 9th level, it has a range of 30 feet for you, and you can cast it as a bonus action.</p>
+		</description>
+		<sheet>
+			<description>You learn the guidance cantrip. Starting at 9th level, it has a range of 30 feet for you, and you can cast it as a bonus action.</description>

```

Comment:
Make the description change at level 9 for the range of 30 feet + bonus action and adjust the wording too.